velox/docs/functions/spark/aggregate.rst

```diff
@@ -24,7 +24,7 @@ General Aggregate Functions
     ``hash`` cannot be null.
     ``numBits`` specifies max capacity of the bloom filter, which allows to trade accuracy for memory.
     In Spark, the value of ``numBits`` is automatically capped at config value 67,108,864.
-    In Velox, the value of ``numBits`` is automatically capped at fixed value 4,194,304.
+    In Velox, the value of ``numBits`` is automatically capped at config value 4,194,304.
```

Comment:
> config value 4,194,304.

That doesn't quite make sense. The value is either fixed (4,194,304) or configurable. Can't be both.

Maybe, "... capped at the value of spark.runtime.bloom_filter.max_num_bits configuration property"